frontends/web/src/routes/settings/manage-accounts.tsx

```diff
@@ -132,20 +163,27 @@ class ManageAccounts extends Component<Props, State> {
         }
         return Object.entries(this.erc20TokenCodes)
             .map(([tokenCode, name]) => {
-                const active = activeTokens && activeTokens.includes(tokenCode);
+                const active = !!activeTokens && activeTokens.includes(tokenCode);
                 return (
                     <div key={tokenCode}
-                        onClick={() => this.toggleToken(ethAccountCode, tokenCode, !active)}
                         className={`${style.token} ${active ? style.tokenActive : style.tokenInactive}`}>
-                        <Logo
-                            active={active}
-                            alt={name}
-                            className={style.tokenIcon}
-                            coinCode={tokenCode}
-                            stacked />
-                        <span className="flex-1 p-left-quarter">
-                            {name} ({tokenCode})
-                        </span>
+                        <div
```

Comment:
why not `<A href...`?

The link is underlined but it also needs a hand/finger mouse pointer.

Reply:
Inactive tokens shouldn't be linked, so we won't always have a link, I'd prefer at the moment to keep using onClick